helm/examples/gce.yaml

```diff
@@ -1,16 +1,9 @@
 common:
-  # Beta PV.NodeAffinity field is used by default. If running against pre-1.10
-  # k8s version, the `useAlphaAPI` flag must be enabled in the configMap.
-  useAlphaAPI: false
   useNodeNameOnly: true
 classes:
 - name: local-scsi
   hostDir: "/mnt/disks/by-uuid/google-local-ssds-scsi-fs"
-  volumeMode: Filesystem
```

Comment:
@msau42 - Why are these being removed?

Reply:
It's filesystem mode by default, so this is redundant.  Are you seeing issues?